Library/Homebrew/cmd/bottle.rb

```diff
@@ -95,10 +95,10 @@ def print_filename(string, filename)
       end
     end
 
-    keg_contains_absolute_symlink_starting_with?(string, keg) || result
+    keg_contains_absolute_symlink_starting_with(string, keg) || result
   end
 
-  def keg_contains_absolute_symlink_starting_with?(string, keg)
+  def keg_contains_absolute_symlink_starting_with(string, keg)
```

Comment:
It does, but I thought it was better fung shui to match style with the existing `keg_contains`. What do you prefer?